src/index.html

```diff
@@ -6,12 +6,253 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta 
+    http-equiv="X-UA-Compatible" 
+    content="ie=edge"
+    >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link 
+    rel="stylesheet" 
+    href="./style.css"
+    >
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      class="form"
+      >
+
+      <fieldset class="group">
+        <legend>
+        Personal information
+        </legend>
+        <div class="simple">
+          <label>
+            Surname:
+            <input 
+            type="text" 
+            name="sname" 
+            required 
+            autocomplete="off"
+            >
+          </label> 
+        </div>
```

Comment:
Please check code styling moments ALL OVER the page. Especially point 2-3
[Recommendations](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
```suggestion
      <fieldset class="group">
        <legend>Personal information</legend>
        <div class="simple">
          <label>
            Surname:
            <input 
              type="text" 
              name="sname" 
              required 
              autocomplete="off"
            >
          </label> 
        </div>
```

Reply:
is not fixed